libpod/options.go

```diff
@@ -194,7 +194,7 @@ func WithDefaultMountsFile(mountsFile string) RuntimeOption {
 			return ErrRuntimeFinalized
 		}
 
-		rt.config.DefaultMountsFile = []string{mountsFile}
+		rt.config.DefaultMountsFile = mountsFile
```

Comment:
Instead of swapping this, we should only be allowing DefaultMountsFile to be set if it's not ""

Reply:
That is happening here right? https://github.com/projectatomic/libpod/blob/master/cmd/podman/libpodruntime/runtime.go#L51

Reply:
We should still do the check here as well